data_finder.py

```diff
@@ -0,0 +1,51 @@
+import twitter
+import pickle
+#who do you want to mimic
+def data_finder(user_handle):
+    while True:
+        #data finder prompst the user to input login material
+        print ('consumer_key:')
+        c_k = input()
+        print ('consumer_secret:')
+        c_s = input()
+        print ('access_token_key:')
+        a_t_k = input()
+        print ('access_token_secret:')
+        a_t_s = input()
+        #data fider uses these to login
```

Comment:
Instead of making an user type API key, it would be simpler to store the API keys in a separate files and referencing them.